pkg/daemon/ceph/osd/daemon.go

```diff
@@ -317,13 +317,6 @@ func getAvailableDevices(context *clusterd.Context, agent *OsdAgent) (*DeviceOsd
 		}
 
 		if device.Type == sys.PartType {
-			// If we detect a partition we have to make sure that ceph-volume will be able to consume it
-			// ceph-volume version 14.2.8 has the right code to support partitions
-			if !agent.clusterInfo.CephVersion.IsAtLeast(cephVolumeRawModeMinCephVersion) {
```

Comment:
I think we still need to find which Octopus version has the right code so we can check that instead of Nautilus